browser/resources/settings/brave_settings_overrides.js

```diff
@@ -1,8 +1,18 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at http://mozilla.org/MPL/2.0/.
 
-window.BravePatching.registerPolymerBehaviors({
+if (!BravePatching) {
+  console.error('BravePatching was not available to brave_settings_overrides.js')
```

Comment:
when would this happen and shouldn't `BravePatching.RegisterPolymerComponentBehaviors...` be in an `else`?

Reply:
It's a helper error to let the developer know they haven't included `polymer_overriding.js`